echopype/convert/api.py

```diff
@@ -248,7 +248,7 @@ def _set_convert_params(param_dict: Dict[str, str]) -> Dict[str, str]:
     out_params["platform_name"] = param_dict.get("platform_name", "")
     out_params["platform_code_ICES"] = param_dict.get("platform_code_ICES", "")
     out_params["platform_type"] = param_dict.get("platform_type", "")
-    out_params["water_level"] = param_dict.get("water_level", None)
+    out_params["range_offset"] = param_dict.get("range_offset", None)
```

Comment:
@leewujung See my comment above about `range_offset` (and previously `water_level`) ever being an attribute rather than a variable. I don't see any reference to such usage in the convention. Is this correct? Is there a known use case where having it as a top-level attribute is far more desirable than having it as a scalar variable in the `Platform` group?

Reply:
I think it is a misnomer. `range_offset` should be a variable indeed. I think it should also remain in the `Platform` group, and don't see a case where it should be in the top-level attribute.

This is related to #581 -- I think we should re-consider what should be in `open_raw` and what should happen during L0-->L1.